addon/templates/components/truncate-multiline.hbs

```diff
@@ -7,6 +7,7 @@
         _inPlace=_buttonInPlace
         onClick=(action "toggleTruncate"))
       isTruncated=truncationState
+      neededTruncating=isTruncated
```

Comment:
We're already yielding this value out. Why would we yield it out twice?

Reply:
Ya, I agree, and thanks for clarifying.

Should we look at renaming those to match the name yielded out?

```js
isTruncated=isTruncated,
neededTruncating=neededTruncating
```